basics/issueCredential.ts

```diff
@@ -0,0 +1,40 @@
+import * as Kilt from "@kiltprotocol/sdk-js";
+import { CType } from "@kiltprotocol/credentials";
+import type {
+  SignerInterface,
+  DidDocument,
+  MultibaseKeyPair,
+} from "@kiltprotocol/types";
+import { Types } from "@kiltprotocol/credentials";
+
+export async function issueCredential(
+  issuerDid: DidDocument,
+  holderDid: DidDocument,
+  signers: SignerInterface[],
+  submitterAccount: MultibaseKeyPair
+): Promise<Types.VerifiableCredential> {
+  const passportCType = await CType.fetchFromChain(
```

Comment:
maybe unpack the ctype here so that you don't have to access `cType` later, that's cleaner semantically.
```suggestion
  const {cType: passportCType} = await CType.fetchFromChain(
```